gidgethub/apps.py

```diff
@@ -0,0 +1,43 @@
+"""Support for GitHub Actions."""
+from typing import cast, Any, Dict
+
+import time
+import jwt
+
+from gidgethub.abc import GitHubAPI
+
+
+def get_jwt(*, app_id: str, private_key: str) -> str:
```

Comment:
In octomachinery, I used to have it as a function but then I implemented an object that represents a private key with fingerprint pinning and it turned out that it's handy to have it as a method:
https://github.com/sanitizers/octomachinery/blob/6b41c1df510f8bbf090ea290941261396e2cb559/octomachinery/github/models/private_key.py